_inc/lib/class-jetpack-podcast-helper.php

```diff
@@ -10,15 +10,58 @@
  */
 class Jetpack_Podcast_Helper {
 	/**
-	 * Gets a list of tracks for the supplied RSS feed. This function is used
-	 * in both server-side block rendering and in API `WPCOM_REST_API_V2_Endpoint_Podcast_Player`.
+	 * Gets podcast data formatted to be used by the Podcast Player block in both server-side
+	 * block rendering and in API `WPCOM_REST_API_V2_Endpoint_Podcast_Player`.
 	 *
 	 * @param string $feed     The RSS feed to load and list tracks for.
 	 * @param int    $quantity Optional. The number of tracks to return.
+	 * @return array|WP_Error The player data or a error object.
+	 */
+	public static function get_player_data( $feed, $quantity = 10 ) {
```

Comment:
Should we remove `$quantity` from here and from the block's PHP side now?

Reply:
The server-side rendering still uses it to only inline data of tracks that will be used. 

We only cut the episode amount in the editor but the component on the frontend displays everything it gets.

Reply:
I'm happy to leave quantity as-is, unless you object.

Reply:
Update: I will be removing quantity in a follow up PR I have started.